README.md

```diff
@@ -1,5 +1,5 @@
 # Thirsty
 
-![Build Status](https://codeship.com/projects/88412/status?branch=master)
+![Build Status](https://codeship.com/projects/0dbdb310-00b0-0133-758f-46d3771abf46/status?branch=master)
```

Comment:
I got this key from my Settings > Webhooks > Codeship, which was what was posted in Slack staff-announcements by Dan. So I'm pretty sure it's the right one.